tests/periph_spi/main.c

```diff
@@ -59,7 +59,7 @@ static struct {
     spi_cs_t cs;
 } spiconf;
 
-char printbuf[64] = {0};
+char printbuf[128] = {0};
```

Comment:
Maybe try making this `SHELL_DEFAULT_BUFSIZE`?

Reply:
Is done.